README.md

```diff
@@ -1,2 +1,3 @@
 # Duck Central
+OOOO
```

Comment:
Might want to make some of these lowercase

Reply:
no